src/protocolstatus.cpp

```diff
@@ -160,7 +160,7 @@ void ProtocolStatus::sendInfo(uint16_t requestedInfo, const std::string& charact
 
 	if (requestedInfo & REQUEST_MISC_SERVER_INFO) {
 		output->addByte(0x12);
-		output->addString(g_config.getString(ConfigManager::MOTD));
+		output->addString("N/A"); //MOTD
```

Comment:
`// MOTD`

Reply:
don't worry, @ranisalt clang will fix that in the future I hope